oio/cli/admin/volume.py

```diff
@@ -219,13 +219,15 @@ def take_action(self, parsed_args):
 
         try:
             all_rawx = self.app.client_manager.admin.rdir_lb.assign_all_rawx(
-                    parsed_args.max_per_rdir)
+                    parsed_args.max_per_rdir,
+                    connection_timeout=30.0, read_timeout=90.0)
         except ClientException as exc:
             if exc.status != 481:
```

Comment:
If exception is due to timeout, we'll hit the raise ?
maybe add a new except OioTimeout ?